mthamayor/auto-mart
# #166257939 Users can mark their advert as sold

#### What does this PR do?
Users can mark their adverts as 'sold'

#### Description of Task to be completed?
Create API endpoint PATCH /car/<:car_id>/status
Unit tests should be written for the endpoint
The endpoint should be tested on Postman

#### How should this be manually tested?
After cloning the repo

Start the development server with
npm run dev
After signing in, and you have an existing purchase order that is still pending
On postman, set the Authorization header with the value
Bearer  <token>
the URL is
http://localhost:3000/api/v1/car/<car_id>/status
The form body should contain the necessary fields as shown below, then hit SEND
![Screenshot (120)](https://user-images.githubusercontent.com/16121915/58424768-12c42080-8090-11e9-9e4f-1fb7e5846ffe.png)


#### Any background context you want to provide?
For the sake of development and tests, the authorization token can be any value

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/166257939
